src/web/routes/application/address/formats.js

```diff
@@ -0,0 +1,15 @@
+const { isString } = require('../../../../common/predicates')
+
+const SINGLE_WORD_REGEX = /\b\w+/g
+
+const toTitleCase = (str) => {
+  return !isString(str) ? str : str.replace(
+    SINGLE_WORD_REGEX,
+    function (txt) { return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase() }
```

Comment:
Might be worth a couple of tests where the String is 1 char long and 0 chars long.